### ISSUE TITLE
In my case

### ISSUE THREAD

**Issue by marcoantonioap** [facebook/react-native#31009]
before:
{
console.log(this.state.dataSource) 

      let movies = this.state.dataSource.map( (val, key) =>{
          console.log(key + val.title)
          return <View key={key}> {val.title} </View>
      });
      
      return (
        <View style={styles.sectionBody}>
          {movies}
        </View>
      );
}

after: I just change the component View of let movies return for a Text component

Hope this helps!

**Comment by markymc** [comment]
@marcoantonioap This is posted as a new issue, but I figure it was intended to be a comment on another issue?

**Comment by chrisbobbe** [comment]
> @marcoantonioap This is posted as a new issue, but I figure it was intended to be a comment on another issue?

That sounds right; I think this issue can be closed.
